### DISCUSSION TITLE
Image placeholders dont parse into top balance list but anywhere else they do

### DISCUSSION THREAD

**Discussion by Verzante** [Zrips/CMI#7959] [Enhancement]
### Description of Issue

Well, title says it, image placeholders like %img_example% dont parse into /baltop list. But anywhere else they do.  I use this placeholder as a currency symbol in config.yml

More information:
- The placeholder itsself is functioning correctly in-game, meaning the problem is not at the side of the placeholder or the provider of it.
- The placeholder comes from ItemsAdder plugin

What are the possible solutions?

1. Fix it so it parses correctly

2. Make a possibility so the [balance] placeholder in lang can be changed so it just displays the amount of money without the currency symbol so I can use image id placeholder (:example:)  instead of regular placeholder. 
Example would be: [balance_nosymbol] 

### Version Information

```shell
https://imgur.com/a/bGEqFYq
```

### Errors

```shell
No errors in console or anywhere else
```

### Relevant Config Sections

```yaml
I will provide here the CurrencySymbol configuration in config.yml:

CurrencySymbol: '&f%img_mince2%'
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by CloudeLecaw** [comment]
the curreny symbol never was made to support a placeholder.

so its more a feature requests for discussion then an issue.
so might better to close the issue and reopen a request for image placeholder support here:
https://github.com/Zrips/CMI/discussions
